docker/ca-cert-injector/Dockerfile

```diff
@@ -0,0 +1,3 @@
+FROM debian:bookworm-slim
```

Comment:
Could this cause problem given the backend image is based on python which is based on ubuntu (.. I think)?

Reply:
I have no idea but I changed to `ubuntu:mantic` to avoid any risks 😉